rswag-specs/lib/rswag/specs.rb

```diff
@@ -9,6 +9,10 @@ module Specs
 
     # Extend RSpec with a swagger-based DSL
     ::RSpec.configure do |c|
+      if c.respond_to?(:disable_monkey_patching!)
+        # Fix "irb: warn: can't alias context from irb_context." in Rails console
+        c.disable_monkey_patching!
```

Comment:
We can be compliant with if someone has the `disable_monkey_patching!` option but it is not our place to set this setting on other project's RSpec config.

This suggestion comes from a good place that I agree with, so the best we could do is leave this commented as a suggestion.

Reply:
Ah that's a good point, I have removed the call to `disable_monkey_patching!`, so this PR just adds `RSpec` in front of the `describe` calls. That still seems to fix the warnings for me.

Reply:
I think I just made a silly mistake. For some reason I assumed this was the spec_helper template that was copied into the project. This is for just fixing this projects specs, I was wrong to ask for this to be removed, and we could add a comment to the template that is copied in suggesting it.

That's what I get for merging things at midnight 😆